metricbeat/docs/modules_list.asciidoc

```diff
@@ -120,8 +120,9 @@ This file is generated! See scripts/mage/docs_collector.go
 |<<metricbeat-metricset-etcd-self,self>>   
 |<<metricbeat-metricset-etcd-store,store>>   
 |<<metricbeat-module-gcp,Google Cloud Platform>>  beta[]   |image:./images/icon-yes.png[Prebuilt dashboards are available]    |  
-.8+| .8+|  |<<metricbeat-metricset-gcp-billing,billing>> beta[]  
+.9+| .9+|  |<<metricbeat-metricset-gcp-billing,billing>> beta[]  
```

Comment:
Not sure if this change is related to this PR. Maybe is possible to remove it if not?

Reply:
That should be because a new metricset was added (dataproc) and the no. of lines increased by 1. The file is automatically updated by running `make docs`.

Reply:
That should be because a new metricset was added (dataproc) and the no. of lines increased by 1. The file is automatically updated by running make docs.